packages/runtime/src/ctx.ts

```diff
@@ -253,7 +253,7 @@ export class Ctx implements Context {
 	 * @param flags asa deployment flags
 	 */
 	deployASA(name: string, fromAccountAddr: AccountAddress, flags: ASADeploymentFlags): ASAInfo {
-		if (this.runtime.loadedAssetsDefs === {}) {
+		if (this.runtime.loadedAssetsDefs === undefined) {
```

Comment:
not sure about this. My IDE warns me that the old expression always returns false